### PR TITLE
SOAP WsClientTest stabilization & MenuIsEnabled condition

### PR DESCRIPTION
**Pull request by theJNOVAK**

### CODE REVIEW

**Comment by apodhrad** [comment]
testPR

**Comment by rawagner** [comment]
testPR

**Comment by apodhrad** [comment]
Please add signoff

**Comment by apodhrad** [comment]
```
[ERROR] Failed to execute goal org.eclipse.tycho:tycho-compiler-plugin:0.26.0:compile (default-compile) on project org.eclipse.reddeer.eclipse: Compilation failure: Compilation failure:
[ERROR] /home/apodhrad/Projects/reddeer/plugins/org.eclipse.reddeer.eclipse/src/org/eclipse/reddeer/eclipse/wst/server/ui/cnf/ServerModule.java:[29]
[ERROR] import org.eclipse.reddeer.swt.condition.MenuIsEnabled;
[ERROR] ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
[ERROR] The import org.eclipse.reddeer.swt.condition.MenuIsEnabled cannot be resolved
[ERROR] /home/apodhrad/Projects/reddeer/plugins/org.eclipse.reddeer.eclipse/src/org/eclipse/reddeer/eclipse/wst/server/ui/cnf/ServerModule.java:[85]
[ERROR] new WaitUntil(new MenuIsEnabled(new ShellMenu("Edit", "Delete")), TimePeriod.LONG);
[ERROR] ^^^^^^^^^^^^^
[ERROR] MenuIsEnabled cannot be resolved to a type
[ERROR] 2 problems (2 errors)
[ERROR] -> [Help 1]
[ERROR] 
```

File: plugins/org.jboss.reddeer.swt/src/org/jboss/reddeer/swt/condition/MenuIsEnabled.java [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+import org.jboss.reddeer.common.condition.AbstractWaitCondition;
+import org.jboss.reddeer.swt.api.Menu;
+
+/**
+ * Condition is met when specified menu is enabled.
+ * 
+ * @author jnovak@redhat.com
+ *
+ */
+public class MenuIsEnabled extends AbstractWaitCondition {
```
Comment (apodhrad) [inline plugins/org.jboss.reddeer.swt/src/org/jboss/reddeer/swt/condition/MenuIsEnabled.java]: Move this class to org.eclipse.reddeer.swt.condition

**Comment by theJNOVAK** [comment]
Class moved and signoff added.

**Comment by theJNOVAK** [comment]
Fixed.

**Comment by apodhrad** [comment]
pushed to master

**Pull request closed without merging**